static/templates/message_edit_form.handlebars

```diff
@@ -5,7 +5,7 @@
     <div class="control-group no-margin">
         <label class="control-label edit-control-label" for="message_edit_topic">{{t "Topic" }}</label>
         <div class="controls edit-controls">
-          <input type="text" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
+          <input type="text" placeholder="{{t "(no topic)" }}" value="{{topic}}" class="message_edit_topic" id="message_edit_topic" />
```

Comment:
I would pass in this value as compose.empty_topic_placeholder() from message_edit.edit_message, in case this "(no topic)" and the compose.empty_topic_placeholder() get translated differently.